stylelint.config.js

```diff
@@ -100,12 +101,15 @@ module.exports = {
     'selector-attribute-brackets-space-inside': 'never',
     'selector-attribute-operator-space-after': 'never',
     'selector-attribute-operator-space-before': 'never',
+    'selector-class-pattern':
+      '^(?:(?:o|c|u|t|s|is|has|_|js|qa)-)?[a-zA-Z0-9]+(?:-[a-zA-Z0-9]+)*(?:__[a-zA-Z0-9]+(?:-[a-zA-Z0-9]+)*)?(?:--[a-zA-Z0-9]+(?:-[a-zA-Z0-9]+)*)?(?:\\[.+\\])?$', // BEM naming convention
```

Comment:
This is an attempt at checking for the BEM naming convention. It's works quite well but we do have css like:

```css
_component {
  .btn {
     width: 100%;
  }
}
```

Which it will pick up on